### ISSUE TITLE
where is nmt_lm_loss used?

### ISSUE THREAD

**Issue by tairan-w** [Mlair77/nmt_adequacy#3]
In the trainer.py code, I see how the loss_add is computed. However, I didn't see where it is used. The line 174 in loss.py the `loss += loss_add` is annotated.
